e2e/support/test_support.go

```diff
@@ -1151,12 +1234,12 @@ func Platform(ns string) func() *v1.IntegrationPlatform {
 					continue
 				}
 				if pl != nil {
-					panic("multiple primary integration platforms found in namespace " + ns)
+					failTest(fmt.Errorf("multiple primary integration platforms found in namespace " + ns))
```

Comment:
should we do like this instead since it's `Errorf`?
```
fmt.Errorf("multiple primary integration platforms found in namespace %q", ns)
```

Reply:
Done